### ISSUE TITLE
'Save message...' folder list is not traversable with arrow keys

### ISSUE THREAD

**Issue by johnnyutahh** [opto/nostalgy-xpi#153]
Sometimes the `Save message [and go there]` pop-up folder list is not traversable with arrow keys. I recall folder-list entries are also not clickable, either (but I'm less certain of this).

In all the same scenarios (described above), once I start re-typing in the folder-search-text-entry area the up/down arrow keys work once again. But... for inital folder-path suggestions (without typing in a new search) the up/down arrow keys (to select one of the initially-suggested folders, or just the single-suggested folder) do not work. Sometimes.

Klaus, does this all make sense?

Maybe this happens when there's only one folder in the suggested list?  Not sure yet. The "broken" behavior seems inconsistent.

I'm currently running Thunderbird v91.5.1, Nostalgy++ v3.2.0.

**Comment by Olifair** [comment]
Yes Johnny, does make sense. This is a problem that has existed since the switch from Nostalgy to Nostalgy-xpi, due to the new add-on standard of TB.  There are some old bug reports on this, but have never been resolved.

**Comment by opto** [comment]
yes, this folder completion and the box is done by TB, not by Nostalgy. I have a similiar experience that 'sometimes, it is strange', but then it self-heals, so I havent done anything about this.
